packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/cancel-deal.jsx

```diff
@@ -43,60 +43,64 @@ const CancelDeal = observer(() => {
     );
 
     return (
-        <Fieldset className='trade-container__fieldset'>
-            <div className='dc-input-wrapper--inline'>
-                {should_show_popover ? (
-                    <Popover
-                        alignment='left'
-                        classNameBubble='trade-container__popover'
-                        is_bubble_hover_enabled
-                        margin={2}
-                        message={
-                            <PopoverMessageCheckbox
-                                defaultChecked={is_do_not_show_selected}
-                                checkboxLabel={localize("Don't show this again")}
-                                message={localize(
-                                    'Take profit and/or stop loss are not available while deal cancellation is active.'
-                                )}
-                                name='should_show_cancellation_warning'
-                                onChange={onPopoverCheckboxChange}
-                            />
-                        }
-                        onBubbleClose={onPopoverClose}
-                        relative_render
-                    >
-                        {input}
-                    </Popover>
-                ) : (
-                    <React.Fragment>{input}</React.Fragment>
-                )}
-                <Popover
-                    alignment='left'
-                    icon='info'
-                    id='dt_cancellation-checkbox__tooltip'
-                    is_bubble_hover_enabled
-                    message={localize(
-                        'Cancel your trade anytime within a chosen time-frame. Triggered automatically if your trade reaches the stop out level within the chosen time-frame.'
+        <React.Fragment>
+            {!!cancellation_range_list.length && (
```

Comment:
Why do we need to wrap everything here with `!!cancellation_range_list.length`?

Reply:
To turn number into a boolean flag. Anything bigger than array with length of 0 is true